app/Page.php

```diff
@@ -112,4 +112,16 @@ public function entityRawQuery($withContent = false)
         $htmlQuery = $withContent ? 'html' : "'' as html";
         return "'BookStack\\\\Page' as entity_type, id, id as entity_id, slug, name, {$this->textField} as text, {$htmlQuery}, book_id, priority, chapter_id, draft, created_by, updated_by, updated_at, created_at";
     }
+
+    /**
+     * Get the current revision for the page if existing
+     * @return \BookStack\PageRevision|null
+     */
+    public function getCurrentRevision()
+    {
+        if ($id = PageRevision::where('page_id', '=', $this->id)->max('id')) {
+            return PageRevision::find($id);
+        }
+        return null;
+    }
```

Comment:
The revision table can contain also contain 'update drafts' (Autosaves) so querying the revisions directly like this may not return what you expect. 

The `$page->revisions` relation (line 62) has the correct filtering applied and is ordered by created_at. Therefore you should be able to simplify this function the following:
```php
return $this->revisions()->first()
```